src/ImageViewer.cpp

```diff
@@ -857,6 +859,27 @@ void ImageViewer::removeImage(shared_ptr<Image> image) {
     }
 }
 
+void ImageViewer::removeAllImages() {
+    if (mImages.empty())
+        return;
+
+    // Reset all focus as a workaround a crash caused by nanogui.
+    // TODO: Remove once a fix exists.
```

Comment:
Interesting... is this crash connected with the one that happens sometimes when closing many images quickly in a row? I've been trying to hunt that one down for a long time.

Reply:
I actually copy-pasted this comment and the `requestFocus()` call from `removeImage` :stuck_out_tongue: 
I don't experience a crash when excluding it, but I thought I shouldn't risk it in case it's a platform-specific issue, and included it anyway.

Reply:
Whoopsie ^_^;